src/components/commons/Buttons.tsx

```diff
@@ -8,20 +8,22 @@ export function Button({
   text,
   className,
   disabled,
+  target,
   href,
   onClick,
 }: {
   text: string;
   className?: string;
   disabled?: boolean;
+  target?: boolean;
```

Comment:
let's use the same type as Link's `target`

Reply:
donee!